### PR TITLE
onion_websocket_write fails properly ...

### PR DESCRIPTION
**Pull request by bstarynk**
when the listen point has no writer, we fails properly.

### CODE REVIEW

File: src/onion/websocket.c [hunk 1]
```diff
@@ -159,7 +159,14 @@ void onion_websocket_free(onion_websocket *ws){
  */
 int onion_websocket_write(onion_websocket* ws, const char* buffer, size_t _len)
 {
-	int len=_len; // I need it singed here
+
+        // signature of writer in listen point.
+        typedef ssize_t (lpwriter_sig_t)(onion_request *req, const char *data, size_t len);
+        lpwriter_sig_t* lpwriter
+	  = ws->req->connection.listen_point->write;
+	int len=_len; // I need it signed here
```
Comment (davidmoreno) [inline src/onion/websocket.c]: Better use ssize_t.

**Comment by davidmoreno** [comment]
There is only that very small issue of using better ssize_t, but I merge anyway.

Thanks!

**Comment by davidmoreno** [comment]
Actually I see the use of int is my fault. I will fix it now.

**Comment by bstarynk** [comment]
You mean on *len*; yes, it should be **size_t** (but that was not the point of my patch). Thanks for merging!

**Pull request merged**